tests/unit/controlgroup/core.js

```diff
@@ -66,4 +65,12 @@ test( "selectmenu: open/close corners", function( assert ) {
 		"vertical: Last selectmenu gets ui-corner-bottom when closed" );
 } );
 
+test( "selectmenu: controlgroupLabel", function( assert ) {
+	expect( 2 );
+	var element = $( ".controlgroup" ).controlgroup();
+	var label = element.find( ".ui-controlgroup-label" );
+	assert.hasClasses( label, "ui-widget ui-widget-content ui-state-default ui-controlgroup-item" );
+	assert.hasClasses( label.find( "span" ), "ui-controlgroup-label-contents" );
+} );
+
```

Comment:
this should also add `controlgroup-label` to `methods.js` to fully test it along with the other widgets in different positions to ensure proper classes. https://github.com/jquery/jquery-ui/blob/master/tests/unit/controlgroup/methods.js#L38